### PR TITLE
Added namespace support, elephant io fluent api and some fixes

### PR DESCRIPTION
**Pull request by fractalzombie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor/
 composer.lock
+.idea
```
Comment (nchaulet) [inline .gitignore]: I do not think we should add this to this module gitignore

File: Service/Client.php [hunk 2]
```diff
@@ -37,13 +37,63 @@ public function __construct(Elephant $elephantIO)
...
+    {
+        $this->elephantIO->initialize();
+
+        return $this;
+    }
+
+    /**
+     * Set namespace for event
+     *
+     * @param $namespace
```
Comment (nchaulet) [inline Service/Client.php]: could you fix the doc format is `@param string $namespace`

File: Service/Client.php [hunk 3]
```diff
@@ -37,13 +37,63 @@ public function __construct(Elephant $elephantIO)
...
+        $this->elephantIO->close();
+    }
+
+    /**
+     * Send to socket.io
+     * @param string $eventName event name
+     * @param mixed  $data      data to send must be serializable
+     * @param string $namespace namespace for event
+     */
+    public function send($eventName, $data, $namespace = '/')
```
Comment (nchaulet) [inline Service/Client.php]: could you set a default vault to null for namespace and use an if condition after

**Review by nchaulet — CHANGES_REQUESTED** [review]

**Comment by nchaulet** [comment]
Thanks for the PR 👍

**Comment by fractalzombie** [comment]
Ok, I will commit changes tomorrow.

**Comment by fractalzombie** [comment]
I Fix issues

**Comment by nchaulet** [comment]
Thanks 👍

**Comment by nchaulet** [comment]
your changes are available in v2.1.0

**Pull request merged**